doc/_gallery/plot_3_1_4_atoms_short_hamming_stft.py

```diff
@@ -21,13 +21,29 @@
 import numpy as np
 import matplotlib.pyplot as plt
 from tftb.generators import atoms
-from scipy.signal import hamming
+from scipy.signal.windows import hamming
 from tftb.processing.linear import ShortTimeFourierTransform
+import matplotlib
+matplotlib.use("TkAgg")
```

Comment:
This might work for you specifically, but not necessarily elsewhere - please avoid hardcoding the backend anywhere.

Reply:
I totally agree. It was simply there to highlight the point in my initial comment above.
I will remove the lines.